apm-server/templates/deployment.yaml

```
@@ -19,8 +19,12 @@ spec:
   template:
     metadata:
       labels:
-        app: apm-server
         release: {{ .Release.Name | quote }}
+        chart: "{{ .Chart.Name }}"
```

```suggestion
        chart: "{{ .Chart.Name }}-{{ .Chart.Version }}"
```

to stay consistent with the other `chart` labels of APM Server chart